astropy/stats/biweight.py

```diff
@@ -4,11 +4,9 @@
 Tukey's biweight function.
 """
 
-
 import numpy as np
 
-from .funcs import median_absolute_deviation
-
+from .funcs import _expand_dims, median_absolute_deviation
```

Comment:
One could also do the numpy version check here:
```
from astropy.utils.compat import NUMPY_LT_1_18

if NUMPY_LT_1_18:
    from .funcs import _expand_dims as np_expand_dims
else:
    from numpy import expand_dims as np_expand_dims
```

Reply:
Added `# NUMPY_LT_1_18` everywhere where `_expand_dims` is used.  See comment below.